src/styles/features/csseditor.css

```diff
@@ -78,34 +78,36 @@ div.pce-overlays > * {
  */
 
 aside .pce-wrapper {
-	--selection-bg: #036dd626;
 	--caret: #ffaa33;
 	--selector: #22a4e6;
-	--property: #55b4d4;
 	--string: #86b300;
 	--keyword: #fa8d3e;
 	--operators: #ed9366;
 }
 
 [data-theme='dark'] aside .pce-wrapper {
 	color-scheme: dark;
+	--selection-bg: #3198ff26;
 	--comment: #888b90bb;
-	--variable: #ddd;
-	--plaintext: #ddd;
+	--variable: #dddddd;
+	--plaintext: #dddddd;
+	--propery: #3ba9ce;
 	--number: #d2a6ff;
 	--color: #98e7cc;
-	--class: #ffc35c;
+	--class: #ee9611;
```

Comment:
You didn't need to change `--class` for the dark theme, but if you prefer this color, you can keep it of course

Reply:
That's definitely an mistake, I prefer the original color!